corehq/apps/cloudcare/static/cloudcare/js/formplayer/menus/api.js

```diff
@@ -149,6 +150,7 @@ hqDefine("cloudcare/js/formplayer/menus/api", function () {
     };
 
     FormplayerFrontend.getChannel().reply("app:select:menus", function (options) {
+        options.selectedValues = (sessionStorage.selectedValues).split(',');
```

Comment:
nit: parens aren't needed

Will `sessionStorage.selectedValues` ever be undefined, and will that make this fail?